example-app/src/main/res/layout/activity_main.xml

```diff
@@ -57,14 +65,11 @@
             android:text="@string/verify"
             android:onClick="onClickVerify" />
 
-        <CheckBox
-            android:id="@+id/webViewDebug"
```

Comment:
Why was this checkbox removed? Isn't there any benefit in keeping the option to allow/disable webview debugging?